src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -40,7 +40,7 @@ internal partial class libgpiod
         /// Get the next gpiochip handle.
         /// </summary>
         /// <param name="iter">The gpiochip iterator object</param>
-        [DllImport(LibgpiodLibrary, SetLastError = true)]
+        [DllImport(LibgpiodLibrary)]
         internal static extern SafeChipHandle gpiod_chip_iter_next(SafeChipIteratorHandle iter);
```

Comment:
Seems this and above methods not used anymore, should we remove it?

Reply:
Yup you are right, I'll remove the unused ones.